en/appendices/5-3-migration-guide.rst

```diff
@@ -58,6 +58,13 @@ ORM
   ``$table->getBehavior('Sluggable')->slugify()`` instead of ``$table->slugify()``.
 - ``EntityTrait::isEmpty()`` is deprecated. Use ``hasValue()`` instead.
 
+Plugin
+------
+
+- Loading of plugins without a plugin class is deprecated. For your existing plugins
+  which don't have one, you can use the ``bin/cake bake plugin MyPlugin --class-only``
+  command, which will create the file ``plugins/MyPlugin/src/MyPlugin.php``.
```

Comment:
Maybe use MyCustom as plugin name
as the filename would then be MyCustomPlugin instead of MyPluginPlugin.